contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -238,30 +238,30 @@ interface ITeleporterMessenger {
      * @return The fee token address and fee amount for a the given message.
      */
     function getFeeInfo(
-        bytes32 destinationChainID,
+        bytes32 destinationBlockchainID,
         uint256 messageID
     ) external view returns (address, uint256);
 
     /**
      * @dev Gets the next message ID to be used for a given chain ID.
      * @return The next message ID to be used to send a message to the given chain ID.
      */
-    function getNextMessageID(bytes32 chainID) external view returns (uint256);
```

Comment:
Can you update this to `destinationBlockchainID` for consistency with the rest of the contract?